.github/workflows/dependency-graph.yml

```diff
@@ -0,0 +1,12 @@
+name: Update Dependency Graph
+on:
+  push:
+    branches:
+      - main # default branch of the project
```

Comment:
should this be 1.0.x?

Reply:
Possibly, I don't know whether `1.0.x` needs to be added to `main`'s `.github/workflows/dependency-graph.yml` or whether we need to change `1.0.x`'s entry. Ill look at this later.